File: backend/src/ai/placement.ts
import { autoPlaceFleet } from "../core/Game.js";
import type { Board } from "../core/Board.js";
import type { Rng } from "../core/rng.js";

/** Place the entire AI fleet randomly on `board`. */
export function placeAiFleet(board: Board, rng: Rng): void {
  autoPlaceFleet(board, rng);
}

File: backend-rs/src/ai/placement.rs
use crate::core::board::Board;
use crate::core::game::auto_place_fleet;
use crate::core::rng::Rng;

pub fn place_ai_fleet(board: &mut Board, rng: &mut Rng) {
    auto_place_fleet(board, rng);
}
